corehq/ex-submodules/couchforms/management/commands/one_off_resave_forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+ATTENTION!
+If you see this file after the year 2015, please delete it immediately.
```

Comment:
Yeah, I made the deletion PR as soon as I submitted this PR.  I think that's gonna be my strategy for these kinds of things where I plan to clean up "after it's over", so I've got a reminder about it and all I have to do is remove the "hold off on merge" flag.